metis-validation/metis-validation-rest/src/main/resources/schema/schematron/schematron-internal.xsl

```diff
@@ -479,13 +464,11 @@
                     <xsl:attribute name="location">
                         <xsl:apply-templates select="." mode="schematron-select-full-path"/>
                     </xsl:attribute>
-                    <svrl:text>
-                        id:
-                        <xsl:text/>
+                    <xsl:attribute name="nodeId">
                         <xsl:value-of select="@rdf:about"/>
-                        <xsl:text/>
-                        An ore:Aggregation must have at least one instance of
-                        edm:rights
+                    </xsl:attribute>
+                    <svrl:text>
+                        An ore:Aggregation must have at least one instance of edm:rights
```

Comment:
Adding it here but it's actually lower:
xsl:template match="ore:Aggregation/edm:provider"
xsl:template match="ore:Aggregation/edm:dataProvider"
I think that rule was not updated, the svrl:text has new line which should be removed.

xsl:template match="*"
Might want to also remove the new lines like the rest.